app/Http/Controllers/LoginController.php

```diff
@@ -141,7 +141,7 @@ public function doDevelopmentLogin()
         $userName = $input[$userId];
 
         // get user club
-        $inputClub = array("bc-Club", "bc-Café");
+        $inputClub = array("bc-Club", "bc-Café", "bd-Club");
```

Comment:
maybe we could use this with the section table
use case: select all section names from the database and pic up one randomly
makes it easier for adding and testing new sections

Reply:
Agreed